### ISSUE TITLE
Ability to change menu colors

### ISSUE THREAD

**Issue by callmevladik** [kubernetes-sigs/headlamp#1401]
I know it refers to bigger story like custom themes capability, but is it possible for now to at least add some identifiers that will help to change selected menu item's colors?

**Comment by joaquimrocha** [comment]
Hey @callmevladik ! Yes, I'd refer to the bigger theme override plans we have.
However, please let me know exactly what you need to apply a diff color to. Is it the sidebar entry (background of an entry)? Is it the icon/text? All of them?

**Comment by callmevladik** [comment]
Hey @joaquimrocha, yeah, I was trying to do the same as in the screenshot hardcoding global css in a weird way which still hasn't give any results as the generated classes are different on production :))
![image](https://github.com/headlamp-k8s/headlamp/assets/42180137/5e8e5ffb-1ea7-46ff-acc7-6a04300c69c2)

![image](https://github.com/headlamp-k8s/headlamp/assets/42180137/d1af5c83-9553-4b52-b2ce-75754b865367).

Also, I was thinking about theme overriding and came to some solution I guess. What about storing themes in localStorage and read them when initializing the ThemeProvider?

**Comment by joaquimrocha** [comment]
Hi @callmevladik . Thanks for your reply and suggestion.
You're suggesting adding it to the localStorage (as opposed, I guess, to keeping it in memory after a plugin sets it up eventually, like e.g. `registerTheme(themeName: string, themeData: Theme)`) is this so we have access to the theme data before the plugins are loaded? Indeed, we'd have to refrain from showing anything with distinctive visual before we load the plugins, but this would have to happen anyway the 1st time we load Headlamp, when there would be nothing in the localStorage set from the plugins.

**Comment by callmevladik** [comment]
@joaquimrocha Yes, I think headlamp can set its theme into localStorage and use it when initializing ThemeProvider, so plugin devs can modify that theme through some headlamp's shipped api like setTheme, resetTheme(to reset theme object to original)

**Comment by illume** [comment]
Is this something you're still interested in @callmevladik ?
